helper/threads.c

```diff
@@ -34,6 +35,38 @@
 
 static odph_helper_options_t helper_options;
 
+static uint8_t get_child_status(int fd)
+{
+	uint8_t status;
+	int ret;
+
+	ret = recv(fd, &status, sizeof(status), MSG_DONTWAIT | MSG_NOSIGNAL);
```

Comment:
`recv` could be wrapped in `TEMP_FAILURE_RETRY`.

Reply:
Added in V2.